pyiode/iode_database/abstract_database.pyx

```diff
@@ -501,6 +508,17 @@ cdef class _AbstractDatabase:
         input_file = str(Path(input_file).resolve())
         self.abstract_db_ptr.merge_into(input_file.encode())
 
+    # GDM>
+    # * I think the API would be nicer if that information was available as properties on the "single object" classes
+    #   (e.g. Equation, Scalar, Table). You might need to create classes for Comment, Identity, List and Variable,
```

Comment:
Not impossible but will require to complexy the implementation of `__getitem__` and `__setitem__`. Not sure the extra work and especialy the future cost maintainance worth to create special Python classes `Comment`, `Identity`, `List` and `Variable` with only one prorperty.